var/spack/repos/builtin/packages/py-keras/package.py

```diff
@@ -50,3 +58,93 @@ class PyKeras(PythonPackage):
     depends_on('py-theano', type=('build', 'run'), when='@:2.2')
     depends_on('py-pyyaml', type=('build', 'run'))
     depends_on('py-six', type=('build', 'run'), when='@:2.2')
+    depends_on('py-tensorflow', type=('build', 'run'), when='@2.5.0:')
+    depends_on('bazel', type='build', when='@2.5.0:')
+    depends_on('protobuf', type='build', when='@2.5.0')
+
+    @when('@2.5.0:')
+    def patch(self):
```

Comment:
I think the readability of the recipe will increase if "patch" for `keras.bzl` is put into a separate file and then `read()` in patch method?